.test-infra/jenkins/job_PerformanceTests_InfluxDBIO_IT.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import CommonJobProperties as common
+import Kubernetes
+
+String jobName = "beam_PerformanceTests_InfluxDBIO_IT"
+
+job(jobName) {
+  common.setTopLevelMainJobProperties(delegate)
+  common.enablePhraseTriggeringFromPullRequest(
+          delegate,
+          'Java InfluxDBIO Performance Test',
+          'Run Java InfluxDBIO Performance Test')
+
+  String namespace = common.getKubernetesNamespace(jobName)
+  String kubeconfigPath = common.getKubeconfigLocationForNamespace(namespace)
+  Kubernetes k8s = Kubernetes.create(delegate, kubeconfigPath, namespace)
+
+  k8s.apply(common.makePathAbsolute("src/.test-infra/kubernetes/influxdb/influxdb.yml"))
+  String influxDBHostName = "LOAD_BALANCER_IP"
+  k8s.loadBalancerIP("influxdb-load-balancer-service", influxDBHostName)
+  Map pipelineOptions = [
+          influxDBURL     : "http://\$${influxDBHostName}:8086",
+          influxDBUserName : "superadmin",
+          influxDBPassword : "supersecretpassword",
+          databaseName : "db1"
+  ]
+
+  steps {
+    gradle {
+      rootBuildScriptDir(common.checkoutDir)
+      common.setGradleSwitches(delegate)
+      switches("--info")
+      switches("-DintegrationTestPipelineOptions=\'${common.joinPipelineOptions(pipelineOptions)}\'")
+      switches("-DintegrationTestRunner=direct")
```

Comment:
I think this should be executed on Dataflow so that we can test integration of a real runner with a real service.
Remember about the appropriate options relevant to the Dataflow runner

Reply:
After looking a bit into the code we have a weird mix of Dataflow/Direct in the runs, do you think there is a way we can do that 'choosable' when running an IT test? (note that I agree with the idea of running on Dataflow, but of course I understand contributors may not have access to it and it is quicker to develop/do it with DirectRunner).